agents-for-bedrock/features-examples/09-fine-grained-access-permissions/README.md

```diff
@@ -0,0 +1,227 @@
+# Designing secure generative AI Application workflows with Amazon Verified Permissions and Agents for Bedrock
+
+[Link AWS Blog: https://aws.amazon.com/blogs/aws/](https://aws.amazon.com/blogs/aws/)
+
+This is sample code we will demonstrate how to design fine-grained access controls using Verified Permissions for a generative AI application that uses agents for Bedrock to answer questions about insurance claims that exist in a claims review system using textual prompts as inputs and outputs.
```

Comment:
add link for https://aws.amazon.com/verified-permissions/

Reply:
added